docs/modules/customize/pages/cookies.adoc

```diff
@@ -0,0 +1,78 @@
+= Cookies
+
+With cookie consent we can explain better what cookies do we use, why we’re using these cookies, and let the user choose which cookies they want to use.
```

Comment:
This is a stylistic change but I checked that we are using the apostrophe `'` in other places of the docs rather than the left/right single quotes, so I think we should follow the same convention here:
```suggestion
With cookie consent we can explain better what cookies do we use, why we're using these cookies, and let the user choose which cookies they want to use.
```